test/e2e/rule_test.go

```diff
@@ -83,6 +103,15 @@ func serializeAlertingConfiguration(t *testing.T, cfg ...alert.AlertmanagerConfi
 	return b
 }
 
+func serializeQueryConfiguration(t *testing.T, cfg ...query.Config) []byte {
+	t.Helper()
+	b, err := yaml.Marshal(&cfg)
+	if err != nil {
```

Comment:
`testutil.Ok` is preferred. Additionally `Errorf` only marks the test as failed but continues the flow, which is wrong here (will cause panics)

Reply:
done